docs/sources/writing-guide/references/index.md

```diff
@@ -14,34 +14,51 @@ keywords:
 
 # Links and cross references
 
-Hugo has built-in shortcodes for creating links to documents.
-The `ref` and `relref` shortcodes display the absolute and relative permalinks to a document, respectively.
+Links are a mechanism to reuse content.
+Instead of writing the same information twice, one can link to a definitive source of truth.
```

Comment:
```suggestion
Links are a mechanism for reusing content.
Instead of writing the same information twice, you can link to a definitive source of truth.
```